Admin/BlockAdmin.php

```diff
@@ -243,7 +241,14 @@ public function getPersistentParameters()
      */
     public function toString($object)
     {
-        return $object->getName();
+        if (!is_object($object)) {
+            return '';
```

Comment:
This is consistent with implementation of toString in parent Admin class of SonataAdmin ...